src/sentry/api/endpoints/organization_dashboard_details.py

```diff
@@ -44,24 +54,28 @@ def _get_dashboard(self, request: Request, organization, dashboard_id):
         return Dashboard.objects.get(id=dashboard_id, organization_id=organization.id)
 
 
+@extend_schema(tags=["Dashboards"])
 @region_silo_endpoint
 class OrganizationDashboardDetailsEndpoint(OrganizationDashboardBase):
     publish_status = {
-        "DELETE": ApiPublishStatus.UNKNOWN,
-        "GET": ApiPublishStatus.UNKNOWN,
-        "PUT": ApiPublishStatus.UNKNOWN,
+        "DELETE": ApiPublishStatus.PUBLIC,
+        "GET": ApiPublishStatus.PUBLIC,
+        "PUT": ApiPublishStatus.PUBLIC,
     }
 
+    @extend_schema(
+        operation_id="Retrieve an Organization's Dashboard",
+        parameters=[GlobalParams.ORG_ID_OR_SLUG, DashboardParams.DASHBOARD_ID],
+        responses={
+            200: DashboardDetailsModelSerializer,
+            403: RESPONSE_FORBIDDEN,
+            404: RESPONSE_NOT_FOUND,
+        },
+        examples=DashboardExamples.DASHBOARD_GET_RESPONSE,
+    )
     def get(self, request: Request, organization, dashboard) -> Response:
         """
-        Retrieve an Organization's Dashboard
-        ````````````````````````````````````
-
         Return details on an individual organization's dashboard.
```

Comment:
noob question: Do customers know what Dashboard is? If it needs explanation can you also add that or link to product docs?

Reply:
I clarified by using the term "Custom Dashboard" in these docs. Customers should know if they're interested in this API, and it is a sidebar item as well